src/ng/http.js

```diff
@@ -34,7 +35,32 @@ function paramSerializerFactory(jQueryMode) {
     });
 
     return parts.length > 0 ? parts.join('&') : '';
-  };
+  }
+
+  function jQueryLikeParamSerializer(params) {
+    if (!params) return '';
+    var parts = [];
+    serialize(params, '', true);
+    return parts.length > 0 ? parts.join('&') : '';
```

Comment:
Minor, but I think this can be just `return parts.join('&');`
In `paramSerializer()` too.